### PR TITLE
Joe add talk to npc

### PR DESCRIPTION
**Pull request by Brookke**

### CODE REVIEW

File: game/core/src/me/lihq/game/people/NPC.java [hunk 1]
```diff
@@ -192,6 +192,16 @@ public NPC setMotive(String motive)
     }
 
     /**
+     * This method sets the NPCs moveability to the value of the parameter
+     *
+     * @param canMove - whether or not to set allow the NPC to move
+     */
+    public void setCanMove(boolean canMove)
```
Comment (Brookke) [inline game/core/src/me/lihq/game/people/NPC.java]: having a getter and setter is a little pointless might as well make it public

Comment (joeShuff) [inline game/core/src/me/lihq/game/people/NPC.java]: Using setters and getters looks nicer though.

Comment (joeShuff) [inline game/core/src/me/lihq/game/people/NPC.java]: and its more secure.

Comment (Brookke) [inline game/core/src/me/lihq/game/people/NPC.java]: secure?

Comment (joeShuff) [inline game/core/src/me/lihq/game/people/NPC.java]: We get to control what the value is set to. Say we had an integer, we can bound it using a setter.
Look at NPC.setKiller(true). That is securing that it only makes them the killer if they arent the victim. If it was public, someone could just do .isKiller = true and then the killer could also be the victim

Comment (Brookke) [inline game/core/src/me/lihq/game/people/NPC.java]: Yeah in that case it makes sense, but when its just setting the value equal to what ever you put it, thats unnecessary

**Pull request merged**